src/lib/instrument.mjs

```
@@ -0,0 +1,7 @@
+import * as Sentry from "@sentry/node";
+Sentry.init({
+    dsn: "https://d42f03df8cc0c949567dcd53627eb909@o4507599063744512.ingest.de.sentry.io/4510328531779664",
```

ESLint

[error] Expected indentation of 2 spaces but found 4.

(style/indent)